powerVCF.psm1

```diff
@@ -2558,28 +2558,24 @@ Export-ModuleMember -Function Get-VCFvRLI
 Function Invoke-VCFCommand {
     <#
         .SYNOPSIS
-        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SOS)
+        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SoS)
     
         .DESCRIPTION
-        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and 
-        subsequently execute elevated SOS commands using the root account. This is why both vcf and root password are
-        mandatory parameters. If passwords are not passed as parameters it will prompt for them.
+        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and subsequently 
+        execute elevated SOS commands using the root account. Bboth vcf and root password are mandatory parameters.
+            If passwords are not passed as parameters it will prompt for them.
     
         .EXAMPLE
-        PS C:\> Connect-VCFCommand -fqdn sfo01vcf01.sfo.rainpole.local -vcfpassword VMware1! -rootPassword VMware1! -sosOption general-health 
+        PS C:\> Invoke-VCFCommand -vcfpassword VMware1! -rootPassword VMware1! -sosOption general-health 
         This example will execute and display the output of "/opt/vmware/sddc-support/sos --general-health" command
     
         .EXAMPLE
-        PS C:\> Connect-VCFCommand -fqdn sfo01vcf01.sfo.rainpole.local -sosOption general-health 
+        PS C:\> Connect-VCFCommand -sosOption general-health 
```

Comment:
Should this be Invoke instead of Connect?

Reply:
that's indeed already been fixed in commit "typos fixed" if you see :-)

Reply:
Ok cool. I thought it would show in the compare if it was part of the PR.